packages/web-components/src/components/cbp-notice/cbp-notice.specs.mdx

```diff
@@ -0,0 +1,29 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Notice/Specifications" />
+
+# cbp-notice
+
```

Comment:
Review for punctuation consistency (add missing periods). I would grab the text for responsiveness and accessibility from another component for consistent wording. This sounds a little clumsy/not as succinct and clear as it could be.

Reply:
update. pulled from cbp-container for the text, can update/refine as needed